examples/controls/README.md

```diff
@@ -0,0 +1,362 @@
+This project is bootstrapped with [Create Elm App](https://github.com/halfzebra/create-elm-app).
```

Comment:
Please remove this documentation, it's not relevant for `elm-aframe` repo, there's an example [here](https://github.com/halfzebra/elm-aframe/tree/master/examples/elm-logo)